appsensor-core/src/main/java/org/owasp/appsensor/core/configuration/server/ServerConfiguration.java

```diff
@@ -190,6 +205,47 @@ public ServerConfiguration setGeolocationDatabasePath(String geolocationDatabase
 		return matches;
 	}
 	
+	/**
+	 * Locate matching client name in custom configuration from server-side config file. 
+	 * 
+	 * @param clientApplicationName the client name for which the detection point exists
+	 */
+	public Boolean findCustomDectectionClientName(String clientName){
```

Comment:
@shreyasdn actually, after looking at it, I'd rather this method be named "hasCustomDetectionPoints()" instead - it makes the intention more clear.